scripts/wrappers/add_token.py

```diff
@@ -120,10 +120,15 @@ def print_yaml(token, check):
 
 def print_short(token, check):
     default_ip, all_ips, port = get_network_info()
-
+    default_addr = str(default_ip) + ":" + str(port) + "/" + str(token) + "/" + str(check)
+    addr_set = set()
+    addr_set.add(default_addr)
     print(f"microk8s join {default_ip}:{port}/{token}/{check}")
     for ip in all_ips:
-        print(f"microk8s join {ip}:{port}/{token}/{check}")
+        addr = str(ip) + ":" + str(port) + "/" + str(token) + "/" + str(check)
+        if addr not in addr_set:
+            print(f"microk8s join {ip}:{port}/{token}/{check}")
+            addr_set.add(addr)
```

Comment:
This is creating a set of IPs we have already "seen". However, the ask here is to simply avoid repeating the default_ip. It should be simpler if you simply check for `ip == default_ip` in the for loop, right?

Reply:
Alright!
I was thinking if the there were duplicate ips in the all_ip list apart from default ip but its much simpler if the default ip is the only problem.
Thanks!